### DISCUSSION TITLE
Support the MathJax cancel extension in GitHub Markdown

### DISCUSSION THREAD

**Discussion by Tech-noneX** [community/community#203499] [Other Feature Feedback, Questions, & Ideas]
### 🏷️ Discussion Type

Product Feedback

### 💬 Feature/Topic Area

Other

### Body

Please enable the MathJax cancel extension in GitHub Markdown.
This extension provides \cancel, \bcancel, \xcancel, and \cancelto. These commands are useful when presenting step-by-step fraction simplification in mathematics, engineering and educational documentation. Enabling the complete extension would also preserve existing Markdown using forms such as \cancel{20}^{4}, avoiding the need to rewrite many calculation blocks. Ideally, GitHub should support both \cancel and \cancelto.
Example:
\cancel{9}^{3}
\require{cancel}
\frac{3}{\cancelto{1}{5}}
\times
\frac{\cancelto{4}{20}}{27}
Currently, GitHub displays \require{cancel} as an unsupported command and ignores \cancelto. Consequently, the replacement values resemble exponents rather than cancelled factors.
Expected behaviour: the factors should have diagonal cancellation marks, with their simplified replacement values displayed beside them.
MathJax already provides this extension:
https://docs.mathjax.org/en/latest/input/tex/extensions/cancel.html
Supporting it would make GitHub Markdown more suitable for readable mathematical and engineering calculations.
<img width="588" height="277" alt="Screenshot 2026-07-30 182529" src="https://github.com/user-attachments/assets/36f7c5fd-6f97-4dff-99a3-46f7c091d4fa" />
<img width="461" height="349" alt="image" src="https://github.com/user-attachments/assets/54d36ce4-b11c-4422-92a5-e3ccd844a74d" />
<img width="524" height="203" alt="image" src="https://github.com/user-attachments/assets/3cd17f15-2212-4eca-b0c2-3dc53be8c961" />

**Comment by github-actions** [comment]
**💬 Your Product Feedback Has Been Submitted 🎉**

Thank you for taking the time to share your insights with us! Your feedback is invaluable as we build a better GitHub experience for all our users.

**Here's what you can expect moving forward ⏩**
- Your input will be carefully reviewed and cataloged by members of our product teams. 
  - Due to the high volume of submissions, we may not always be able to provide individual responses.
  - Rest assured, your feedback will help chart our course for product improvements.
- Other users may engage with your post, sharing their own perspectives or experiences. 
- GitHub staff may reach out for further clarification or insight. 
  - We may 'Answer' your discussion if there is a current solution, workaround, or roadmap/changelog post related to the feedback.

**Where to look to see what's shipping 👀**
- Read the [Changelog](https://github.blog/changelog/) for real-time updates on the latest GitHub features, enhancements, and calls for feedback.
- Explore our [Product Roadmap](https://github.com/orgs/github/projects/4247), which details upcoming major releases and initiatives.

**What you can do in the meantime 💻**
- Upvote and comment on other user feedback Discussions that resonate with you.
- Add more information at any point! Useful details include: use cases, relevant labels, desired outcomes, and any accompanying screenshots.

As a member of the GitHub community, your participation is essential. While we can't promise that every suggestion will be implemented, we want to emphasize that your feedback is instrumental in guiding our decisions and priorities.

Thank you once again for your contribution to making GitHub even better! We're grateful for your ongoing support and collaboration in shaping the future of our platform. ⭐

**Comment by mohammadrezwankhan** [comment]


**Reply by Tech-noneX** [reply]
Thank you for the clear explanation. This confirms that the problem is GitHub’s MathJax configuration rather than the TeX syntax. I would like to keep this discussion open as a feature request for GitHub to allowlist the MathJax cancel extension. It would be helpful if a member of GitHub staff could confirm whether this extension could be considered for support.
